src/poliastro/tests/tests_twobody/test_perturbations.py

```diff
@@ -31,3 +32,35 @@ def test_J2_propagation_Earth():
 
     assert_quantity_allclose(raan_variation_rate, -0.172 * u.deg / u.h, rtol=1e-2)
     assert_quantity_allclose(argp_variation_rate, 0.282 * u.deg / u.h, rtol=1e-2)
+
+
+def test_atmospheric_drag():
+    # http://farside.ph.utexas.edu/teaching/celestial/Celestialhtml/node94.html#sair (10.148)
+    # given the expression for \dot{a} / a, aproximate \Delta a \approx F_a * \Delta t
+
+    R = Earth.R.to(u.km).value
+    k = Earth.k.to(u.km**3 / u.s**2).value
+
+    # parameters of a circular orbit with h = 10 km
+    h = 250  # km
+    r0 = np.array([R + h, 0, 0])  # km
+    v0 = np.array([0, np.sqrt(k / (R + h)), 0])  # km/s
+
+    # parameters of a body
+    C_D = 2.2  # dimentionless
+    A = ((np.pi / 4.0) * (u.m**2)).to(u.km**2).value  # km^2
+    m = 100  # kg
+    B = C_D * A / m
```

Comment:
What do you think as having C_D and m as input instead of B? So we compute B inside the function